generate_star_index.wdl

```diff
@@ -2,6 +2,8 @@ version 1.0
 
 # ENCODE micro rna seq pipeline: generate star index
 
+import "wdl/structs/runtime.wdl"
+
 workflow generate_STAR_index {
     meta {
         author: "Otto Jolanki"
```

Comment:
You can add `description` to workflow's `meta` to show it on dockstore's UI.
This should be helpful for users to find an example input JSON to run a pipeline on Terra/Anvil/DX.
```
workflow atac {
    meta {
        author: 'Jin wook Lee'
        email: 'leepc12@gmail.com'
        description: 'ENCODE ATAC-Seq pipeline. See https://github.com/ENCODE-DCC/atac-seq-pipeline for more details. e.g. example input JSON for Terra/Anvil.'
    }
}
```

Reply:
Thanks. I'll add those in the dockstore specific PR.